run.sh

```diff
@@ -11,7 +11,8 @@ elif [ -n "${INIT_RESTORE_LATEST}" ]; then
       echo "waiting database container..."
       sleep 1
   done
-  find /backup -maxdepth 1 -name '*.sql.gz' | tail -1 | xargs /restore.sh
+  # The [^l] is needed to exclude the latest backup file, and sort only data-tagged backups
+  find /backup -maxdepth 1 -name '[^l]*.sql.gz' | sort | tail -1 | xargs /restore.sh
```

Comment:
I think it is not clear enough why this works. Could you add the full "latest" word in the glob expression? I guess this is what you are doing, excluding the files starting with an "l" from the sort.

Reply:
I confirm the code is excluding the files containing the latest word, and changing the syntax could make it clear, without influence output of code.

What do you think about do explicit reference to date-format?

```
# Needed to exclude the 'latest.<database>.sql.gz' file, consider only filenames starting with number
# Only data-tagged backups, eg. '202212250457.database.sql.gz', must be trapped by the regex
find /backup -maxdepth 1 -name '[0-9]*.*.sql.gz' | sort | tail -1 | xargs /restore.sh
```

If you think that could confuse more, than it is better to only change with `[^latest]`

Reply:
I thinks date-format is even better, thanks!

Reply:
Done!